### DISCUSSION TITLE
Android S / 12

### DISCUSSION THREAD

**Discussion by roxteddy** [ionic-team/capacitor#4339] [General]
Hello everyone !

I could not find any topic for the new Android API so here it is.

So I tried to launch my capacitor (2.4.6) app on a virtual Pixel 4 running Android 12 (API S) and it did not work. The app is stuck on the splash screen.
The splash screen is set with timeout 0 and I hide it on platform ready. Platform ready occurs and I'm able to see the login page from the Chrome's inspector.
Before doing more investigation, I would like to know if anyone was able to run their app properly on this API.

Thanks in avance for your input ^^

**Comment by jcesarmobile** [comment]
I've just created a new app and tested on Android S emulator and works without problems.

White screens are usually caused by javascript errors, you should check the errors on Chrome remote debugger.
